system-addon/content-src/components/ContextMenu/_ContextMenu.scss

```diff
@@ -1,5 +1,5 @@
 .context-menu {
-  background: $background-primary;
+  background: var(--newtab-background-color);
   border-radius: $context-menu-border-radius;
   box-shadow: $context-menu-shadow;
```

Comment:
This context menu box-shadow / border should probably be a themed value. Not entirely sure what it should be.. At least the second 1px shadow should be something closer to --newtab-border-primary-color.

--newtab-border-primary-color ![screen shot 2018-03-23 at 5 46 47 pm](https://user-images.githubusercontent.com/438537/37858486-55223a3c-2ec2-11e8-8769-e9729928bd74.png)

current shadow ![screen shot 2018-03-23 at 5 42 44 pm](https://user-images.githubusercontent.com/438537/37858490-5c9eed00-2ec2-11e8-8c75-5dd14ea68f81.png)